common/src/main/java/org/dash/wallet/common/ui/dialogs/AdaptiveDialog.kt

```diff
@@ -188,16 +187,6 @@ open class AdaptiveDialog(@LayoutRes private val layout: Int): DialogFragment()
             }
         }
 
-        if (negativeButtonSecondary != null && !isPositiveButtonShown) {
```

Comment:
Since the design of dialogs is changed, the secondary style for the negative button is not needed anymore.